repose-aggregator/src/docs/asciidoc/services/index.adoc

```diff
@@ -15,11 +15,11 @@ Enables the local Repose node or an entire cluster to store various types of dat
 The HTTP Connection Pool service is used for all outbound communication, and allows you to specify a number of things about those connections.
 
 == <<metrics.adoc#, Metrics Service>>
-Enables the registration, collection, and reporting of metrics across Repose.
-Such metrics provide insight into the state of Repose and its components at any given time.
+Enables the registration, collection, and reporting of metrics across *Repose*.
```

Comment:
Who wrote this glorious line?! So concise. So beautiful.

Reply:
:thinking: